src/wp-includes/theme.php

```diff
@@ -4181,7 +4181,11 @@ function _add_default_theme_supports() {
 	add_theme_support( 'post-thumbnails' );
 	add_theme_support( 'responsive-embeds' );
 	add_theme_support( 'editor-styles' );
-	add_theme_support( 'html5', array( 'comment-form', 'comment-list', 'style', 'script' ) );
+	/*
+	 * Make block themes support HTML5 by default for all non-block content.
+	 * Blocks contain their own markup, which is HTML5.
```

Comment:
>The comment is slightly incorrect as the comment blocks are using the default template functions. That's also why they are part of the current list and nothing else beside script/style.

Aha. You're right. Let's improve the inline comment. How about this? @ocean90 

```suggestion
	 * Makes block themes support HTML5 by default for the comment block and search form
	 * ( the default template functions) and `[caption]` and `[gallery]` shortcodes. 
	 * Other blocks contain their own markup, which is HTML5.
```